packages/fxa-content-server/app/scripts/templates/sign_up_password.mustache

```diff
@@ -19,16 +19,20 @@
       </div>
       {{/canChangeAccount}}
 
-      <div class="input-row password-row">
-        <input name="password" id="password" type="password" class="password check-password tooltip-below" placeholder="{{#t}}Password{{/t}}" value="{{ password }}" pattern=".{8,}" required autofocus data-form-prefill="password" data-synchronize-show="true" />
-        <div class="helper-balloon" tabindex="-1"></div>
+      <div class="tooltip-container mb-5">
+        <input name="password" id="password" type="password" class="input-text tooltip-below" placeholder="{{#t}}Password{{/t}}" value="{{ password }}" pattern=".{8,}" required autofocus data-form-prefill="password" data-synchronize-show="true" />
+        <div id="password-strength-balloon-container" tabindex="-1"></div>
       </div>
 
-      <div class="input-row password-row">
-        <input id="vpassword" type="password" class="password check-password tooltip-below" placeholder="{{#t}}Repeat password{{/t}}" pattern=".{8,}" required data-synchronize-show="true" />
+      <div class="tooltip-container mb-5">
+        <input id="vpassword" type="password" class="input-text tooltip-below" placeholder="{{#t}}Repeat password{{/t}}" pattern=".{8,}" required data-synchronize-show="true" />
 
-        <div class="input-help input-help-focused input-help-signup input-help-balloon">
-          {{#unsafeTranslate}}You need this password to access any encrypted data you store with us.<br/>A reset means potentially losing data like passwords and bookmarks.{{/unsafeTranslate}}
+        <div class="input-balloon hidden" id="vpassword-balloon">
+          <div class="before:content-key flex before:w-8">
+            <p class="ltr:pl-2 rtl:pr-2">
+              {{#unsafeTranslate}}You need this password to access any encrypted data you store with us.<br class="mb-3">A reset means potentially losing data like passwords and bookmarks.{{/unsafeTranslate}}
+            </p>
```

Comment:
I think for something like this we could search and replace that tag in teh translated strings in the l10n repo.

Reply:
Good call, I'll do that today.